optuna/storages/_rdb/storage.py

```diff
@@ -121,18 +127,31 @@ def objective(trial):
     .. _pool_pre_ping:
         https://docs.sqlalchemy.org/en/13/core/engines.html#sqlalchemy.create_engine.params.
         pool_pre_ping
+
+    Raises:
+        :exc:`ValueError`:
+            If the given `heartbeat_interval` or `grace_period` is not a positive integer.
     """
```

Comment:
Could you add a note that a process that was failed by heartbeat but was actually running will raise RuntimeError?

Reply:
Others look good to me.

Reply:
Thanks a lot! Fixed.